layers/ntp.go

```diff
@@ -0,0 +1,450 @@
+//******************************************************************************
+//
+// Copyright 2016 Ross N. Williams. All rights reserved.
```

Comment:
Use the standard copyright from the other files:

// Copyright 2012 Google, Inc. All rights reserved.
//
// Use of this source code is governed by a BSD-style license
// that can be found in the LICENSE file in the root of the source
// tree.